app/controllers/katello/api/v2/capsules_controller.rb

```diff
@@ -31,5 +33,13 @@ def resource_class
     def authorized
       User.current.allowed_to?(params.slice(:action, :id).merge(controller: 'api/v2/smart_proxies'))
     end
+
+    # By default, Foreman required users to be admin or have the "view_capsule" permission.
```

Comment:
Maybe something like this would be more clear:
```suggestion
    # Without this method, Foreman required non-admin users to have the non-existent "view_capsule" permission.
```